src/verbosity.jl

```diff
@@ -7,7 +7,15 @@ function restore_callsite_source_position!(expr, src)
     # Logging.jl macros; otherwise, they would always report this (verbosity.jl)
     # file as the logging callsite
     expr.args[1].args[2] = src
-    return expr
+    ex = quote
+        LoggingExtras.deprecate_verbosity($(Meta.quot(m)))
+        $expr
+    end
+    return ex
+end
+
+function deprecate_verbosity(m)
+    Base.depwarn("Verbosity logging macros are deprecated as they are not compatible with juliac-compiled programs", m; force=true)
```

Comment:
```suggestion
    Base.depwarn("Verbosity logging macros are deprecated as they are not compatible with juliac-compiled programs", m)
```

I see no reason to `force`.

Reply:
Ah yes, I was just testing that the depwarn actually worked. Good catch.